pyslackersweb/tasks.py

```diff
@@ -103,8 +116,62 @@ def sync_slack_timezones(app: web.Application):
                 slack_timezones=dict(counter.most_common(100)),
                 slack_user_count=sum(counter.values()),
             )
-        except Exception:
+        except Exception:  # pylint: disable=broad-except
             logger.exception("Error refreshing slack user cache")
-            raise
+            return
```

Comment:
I don't raise here because nothing else will catch it and `asyncio` will complains that we have an unhandled exception